jdk/src/share/classes/java/lang/invoke/MethodHandles.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2008, 2016, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2008, 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
https://github.com/ibmruntimes/openj9-openjdk-jdk/commit/ab6b9c35c7541ea628b028cb1357b904a72de5d9 has a 2021 end date.

Reply:
Updated to match the 2021 end date.